packages/neuron-wallet/src/block-sync-renderer/sync/indexer-connector.ts

```diff
@@ -117,7 +117,10 @@ export default class IndexerConnector {
 
     while (this.pollingIndexer) {
       const indexerTipBlock = await this.indexer.tip()
-      await this.synchronize(indexerTipBlock)
+      await this.synchronize({
+        blockHash: indexerTipBlock.blockHash,
+        blockNumber: indexerTipBlock.blockNumber
+      })
```

Comment:
What's the difference between

```typescript
this.synchronize(indexerTipBlock)
```


and
```typescript
this.synchronize({
  blockHash: indexerTipBlock.blockHash,
  blockNumber: indexerTipBlock.blockNumber,
})
```

Reply:
they are the same, and this is just to keep the changes as small as possible

Reply:
But the original code block is

```typescript
await this.synchronize(indexerTipBlock)
```

Which can be left unchanged.

Reply:
refactored